<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tables</class>
 <widget class="QDialog" name="Tables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Book</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>2</y>
         <width>481</width>
         <height>411</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="labelBookTypeId_2">
            <property name="text">
             <string>TypeId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBookTypeId_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="labelBookName_2">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBookName_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="labelBookPageCount_2">
            <property name="text">
             <string>Page count</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditBookPageCount_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="labelBookPrice_2">
            <property name="text">
             <string>Price</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditBookPrice_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="labelBookAuthorId_2">
            <property name="text">
             <string>AuthorId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBookAuthorId_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="labelBookPublisherId_2">
            <property name="text">
             <string>PublisherId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBookPublisherId_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="labelBookAmount_2">
            <property name="text">
             <string>Amount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditBookAmount_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableViewBooks">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>-10</y>
         <width>771</width>
         <height>661</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>430</y>
         <width>481</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QPushButton" name="pushButtonBookAdd_2">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBookEdit_2">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBookDelete_2">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButtonLoadDataBooks">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>610</y>
         <width>471</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Load data</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Student</string>
      </attribute>
      <widget class="QTableView" name="tableViewStudents">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>0</y>
         <width>771</width>
         <height>661</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>471</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <widget class="QPushButton" name="pushButtonStudentAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStudentEdit">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStudentDelete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButtonStudentLoadData">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>620</y>
         <width>471</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Load data</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>20</y>
         <width>471</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="labelStudentStudentId">
            <property name="text">
             <string>StudentId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxStudentStudentId">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="labelStudentFirstName">
            <property name="text">
             <string>First name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStudentFirstName"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="labelStudentSurname">
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStudentSurname"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="labelStudentCourse">
            <property name="text">
             <string>Course</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxStudentCourse">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="labelStudentPhone">
            <property name="text">
             <string>Phone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStudentPhone"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="labelStudentBirthDate">
            <property name="text">
             <string>Birth date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditStudentBirthDate">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="displayFormat">
             <string>yyyy.MM.dd</string>
            </property>
            <property name="timeSpec">
             <enum>Qt::LocalTime</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QLabel" name="labelStudentSex">
            <property name="text">
             <string>Sex</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxStudentSex">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Borrows</string>
      </attribute>
      <widget class="QTableView" name="tableViewBorrows">
       <property name="geometry">
        <rect>
         <x>493</x>
         <y>0</y>
         <width>771</width>
         <height>661</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>471</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_121">
          <item>
           <widget class="QLabel" name="labelBorrowsBorrowId">
            <property name="text">
             <string>BorrowId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBorrowsBorrowId">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_122">
          <item>
           <widget class="QLabel" name="labelBorrowsStudentId">
            <property name="text">
             <string>StudentId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBorrowsStudentId">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_123">
          <item>
           <widget class="QLabel" name="labelBorrowsBookId">
            <property name="text">
             <string>BookId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBorrowsBookId">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelBorrowsEmployeeId">
            <property name="text">
             <string>EmployeeId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBorrowsEmployeeId">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_126">
          <item>
           <widget class="QLabel" name="labelBorrowsTakenDate">
            <property name="text">
             <string>Taken date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditBorrowsTakenDate">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="displayFormat">
             <string>yyyy.MM.dd</string>
            </property>
            <property name="timeSpec">
             <enum>Qt::LocalTime</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_127">
          <item>
           <widget class="QLabel" name="labelBorrowsBroughtDate">
            <property name="text">
             <string>Brought date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEditBorrowsBroughtDate">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="displayFormat">
             <string>yyyy.MM.dd</string>
            </property>
            <property name="timeSpec">
             <enum>Qt::LocalTime</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_25">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>360</y>
         <width>481</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_128">
        <item>
         <widget class="QPushButton" name="pushButtonBorrowsGiveBook">
          <property name="text">
           <string>Give book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBorrowsReturnBook">
          <property name="text">
           <string>Return book</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButtonBorrowsLoadData">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>620</y>
         <width>471</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Load data</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_13">
      <attribute name="title">
       <string>Employee</string>
      </attribute>
      <widget class="QPushButton" name="pushButtonStudentLoadData_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>610</y>
         <width>471</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Load data</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableViewStudents_4">
       <property name="geometry">
        <rect>
         <x>493</x>
         <y>10</y>
         <width>771</width>
         <height>661</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>471</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_57">
        <item>
         <widget class="QPushButton" name="pushButtonStudentAdd_4">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStudentEdit_4">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStudentDelete_4">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>471</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_58">
          <item>
           <widget class="QLabel" name="labelStudentStudentId_4">
            <property name="text">
             <string>EmployeeId</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxStudentStudentId_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_59">
          <item>
           <widget class="QLabel" name="labelStudentFirstName_4">
            <property name="text">
             <string>First name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStudentFirstName_4"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_60">
          <item>
           <widget class="QLabel" name="labelStudentSurname_4">
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStudentSurname_4"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_61">
          <item>
           <widget class="QLabel" name="labelStudentCourse_4">
            <property name="text">
             <string>Course</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxStudentCourse_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_62">
          <item>
           <widget class="QLabel" name="labelStudentPhone_4">
            <property name="text">
             <string>Phone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditStudentPhone_4"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_14">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_15">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
